ols/utils/errors_parsing.py

```diff
@@ -7,10 +7,24 @@
 from ibm_watsonx_ai.wml_client_error import ApiRequestFailure
 from openai import BadRequestError
 
+from ols import config
+
 # Constants for default messages and status codes
 DEFAULT_ERROR_MESSAGE = "An error occurred during LLM invocation. Please contact your OpenShift Lightspeed administrator."  # noqa: E501
 DEFAULT_STATUS_CODE = status.HTTP_500_INTERNAL_SERVER_ERROR
 
+PROMPT_TOO_LONG_WITH_TOOLS_ERROR_MSG = (
+    "The request exceeds the allowable token limit due to tool usage. "
+    "Please refine your prompt to be more specific or reduce its length."
```

Comment:
@asamal4 The error msg is not the same as in the https://issues.redhat.com/browse/OLS-1854, let me know if you want to have different one.

Reply:
updated